Fixes to style guide, design (sumo.scss), README.md (#4132)
SumoLogic/sumologic-documentation: README.md

## Contributing Content

> For detailed instructions, including our style guide, see [Contributor Guidelines](/docs/contributing).

Author: ```suggestion
For detailed instructions, including our style guide, see [Contributor Guidelines](https://help.sumologic.com/docs/contributing).
```

> For detailed instructions, including our style guide, see [Contributor Guidelines](https://help.sumologic.com/docs/contributing).

Author: Needs to be the absolute link - this is the README

> We recommend forking our repo, applying your content changes in a new branch, and submitting a pull request. We will work with you to get this content reviewed, tested, and merged for publishing.

Author: ```suggestion
We recommend you fork our repo, create a new branch, apply your content changes to that branch, and then submit a pull request. We will work with you to get this content reviewed, tested, and merged for publishing.
```

> We recommend you fork our repo, create a new branch, apply your content changes to that branch, and then submit a pull request. We will work with you to get this content reviewed, tested, and merged for publishing.

Author: ```suggestion
We recommend forking our repo, creating a new branch for your content changes, and submitting a pull request. We will help review, test, and merge the content for publishing.
```
```suggestion
We recommend you fork our repo, create a new branch, apply your content changes to that branch, and then submit a pull request. We will work with you to get this content reviewed, tested, and merged for publishing.
```
